cli/tests/end_to_end/mod.rs

```diff
@@ -0,0 +1,67 @@
+use std::path::{Path, PathBuf};
+use std::{env, fs};
+
+use phylum_cli::api::{PhylumApi, PhylumApiError, ResponseError};
+use phylum_cli::config::{AuthInfo, Config, ConnectionInfo};
+use phylum_types::types::auth::RefreshToken;
+use reqwest::StatusCode;
+
+mod extension;
+
+// TODO: Hardcode this?
+const API_URL: &str = "https://api.staging.phylum.io";
+const PROJECT_NAME: &str = "integration-tests";
+
+/// Create config file for the desired environment.
+pub fn create_config(dir: &Path) -> PathBuf {
```

Comment:
I'd also add this as a component of the builder mentioned above.

Reply:
Indeed -- we probably want overall improvements to the structure of the test helpers, but I'd say this is fine for now.